### PR TITLE
chore(KFLUXINFRA-743) Prod local platforms

### PR DESCRIPTION
**Pull request by ifireball**
Adding a set of MPC platforms that use the "local" definitions.
Also adding "mlarge" platforms as a migration path away from using MPC with generic platform names as a way to run on a VM.

This change would require #4328 to work.

### CODE REVIEW

**Review by arewm — APPROVED** [review]
This looks good, but I think it actually requires #4328 as a pre-requisite and not #4327.

**Comment by ifireball** [comment]
> This looks good, but I think it actually requires #4328 as a pre-requisite and not #4327.

What? A typo? I don't make any typos!

**Review by ifireball — COMMENTED** [review]
@hugares

> @ifireball Can you rebase the PR, it has conflicts 

Done. Please re-review.

**Review by hugares — APPROVED** [review]
/lgtm

**Comment by hugares** [comment]
@ifireball Can I remove the hold label, are we ready to merge this one?

**Comment by ifireball** [comment]
> @ifireball Can I remove the hold label, are we ready to merge this one?

I didn't set it (not sure who did, @arewm ?), so I don't mind removing it

**Comment by hugares** [comment]
> I didn't set it (not sure who did, @arewm ?), so I don't mind removing it

I assume @arewm set it to not merge this PR before the one it depended on was merged (https://github.com/redhat-appstudio/infra-deployments/pull/4328)

I removed the hold label

**Comment by arewm** [comment]
Yep, that's the reason I added it. Thanks for removing it!

**Pull request merged**
